BD-5431 Update Bluepoint SDK with v16.0.0

nklymenko opened the pull request.

Commit:
BD-5431 Upgrade Braze SDK

Commit:
BD-5431 Upgrade Braze SDK and Bluedot SDK to v15.6.1

Commit:
BD-5431 Add a stub API Keys

Commit:
BD-5431 Update BluePoint SDK to v16.0.0

Commit:
BD-5431 Add services for Braze integration

Commit:
BD-5431 Update with SDK 16.0.0

nehabluedot commented:
@nklymenko SDK is not initialising after starting the App. Looks like some existing issue, It needs some investigation.
I am looking into it.

nehabluedot committed:
- updated to gradle 8
- Fixed Initialization Error
- Fixed Zone_id and fence_id type error for Braze event logging
- Removed unused code

nehabluedot committed:
Merge pull request #6 from Bluedot-Innovation/ni/fix-init

- updated to gradle 8 and fixed initialization error
